src/routes/(console)/organization-[organization]/+layout.ts

```diff
@@ -65,9 +59,22 @@ export const load: LayoutLoad = async ({ params, depends }) => {
             scopes
         };
     } catch (e) {
-        const prefs = await sdk.forConsole.account.getPrefs();
         const newPrefs = { ...prefs, organization: null };
-        sdk.forConsole.account.updatePrefs(newPrefs);
+        void sdk.forConsole.account.updatePrefs(newPrefs);
         error(e.code, e.message);
     }
 };
+
+// load the invoice and add a banner in bg
+function loadFailedInvoices(teamId: string) {
+    failedInvoice.load(teamId).then(() => {
+        if (get(failedInvoice)) {
+            headerAlert.add({
+                show: true,
+                component: ProjectsAtRisk,
+                id: 'projectsAtRisk',
+                importance: 1
+            });
+        }
+    });
```

Comment:
this function is a bit weird, we load the invoice and then `get` the store. why dont we just get it in the promise chain?

Reply:
not removing this atm, the function has references across multiple components. I'll relay this to billing in the meantime. this feels a bit out of scope for this one.